### DISCUSSION TITLE
Workload Config Error when creating a sample item in the frontend sample

### DISCUSSION THREAD

**Discussion by JosEilersTwyzer** [microsoft/Microsoft-Fabric-workload-development-sample#45] [Help] [answered]
Hi,

I was following the tutorial part 'Set up authentication', however when following the part **Ask for a token and consent the application**
I end up with the following error:

![image](https://github.com/user-attachments/assets/d6802474-1a73-4c1a-bb97-832dea72672f)

I triple checked all the configurations in the frontend but don't see any mistakes, what else could I check for this one?

**Comment by ramizsha** [comment]
@JosEilersTwyzer please make sure that:
1. Your audience is in this format: api://localdevinstance/[the tenant id of the signed in user in fabric]/[your workload name]/[optional string]
2. Your redirect URI starts with http://localhost.

Did you add your configurations in .env.dev file? https://github.com/microsoft/Microsoft-Fabric-workload-development-sample/blob/main/Frontend/.env.dev

**Reply by JosEilersTwyzer** [reply, marked as answer]
@ramizsha : This was due to the fact I was running locally, but didnt have the correct naming for the workload in the manifest and other configuration files (both backend, frontend and in the application registration). It should be named 'Org.{workloadname}' however this is not documented yet it looks like (also not clear from the example configs). Can this be updated? It is working now.

**Reply by ramizsha** [reply]
We are working on adding this to our docs.
